Quaver.Shared/Screens/Settings/Elements/SettingsScrollDirection4K.cs

```diff
@@ -0,0 +1,53 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. 
+ * Copyright (c) 2017-2018 Swan & The Quaver Team <support@quavergame.com>.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Quaver.Shared.Config;
+using Wobble.Bindables;
+
+namespace Quaver.Shared.Screens.Settings.Elements
+{
+    public class SettingsScrollDirection4K : SettingsHorizontalSelector
```

Comment:
There is no need to duplicate this code for both 4k and 7k if it's doing the same thing.